public/css/zipkin.css

```diff
@@ -51,3 +51,11 @@ table.wide-table td {
 table.wide-table th {
   font-size: 0.8em;
 }
+
+p.disclaimer {
+  font-size: 0.8em;
+}
+
+p.notice {
+  font-size: 0.8em;
+}
```

Comment:
nits: new line at the end

Reply:
We can update the `.editorconfig` file to enforce this (as long as people are using editors that understand and apply editorconfig rules). We can do it in a separate PR, though.
https://github.com/apache/incubator-zipkin-website/blob/master/.editorconfig